pkgs/by-name/ka/kaf/package.nix

```diff
@@ -20,6 +21,13 @@ buildGoModule rec {
   # Many tests require a running Kafka instance
   doCheck = false;
 
+  postInstall = lib.optionalString (!stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
```

Comment:
The other way around (i.e. drop the `!`) - we should only try to run the `kaf` binary if we can execute it.